overwatch/processing/detectors/TPC.py

```diff
@@ -20,137 +20,39 @@
 # Used for sorting and generating html
 from .. import processingClasses
 
-class TPCTrendingObjectMean(processingClasses.trendingObject):
-    """ Basic trending object which extracts the mean from a 1D histogram.
+from overwatch.processing.trending.info import TrendingInfo, TrendingInfoException
+from overwatch.processing.trending.objects import mean, maximum, stdDev
```

Comment:
I considered such approach, but as I know there is a problem. When we have import like
`import overwatch.processing.trending.objects as trendingObjects`
we have imported package "overwatch.processing.trending.objects", but it is impossible to refer to any module in this package, because none of those modules were imported by package in __init__.py

So we can add in __init__.py:
`from .mean import MeanTrending`
and it will be ok.

But if there will be also (for example) 30 other trendingObject, we must import all of them. If we need only one, it would be inefficient.

Eventually I decided to change it, because those names are very common, so if there were 30 objects, all of them would pollute spacename.

Reply:
That's a good point. In that case, I would probably advocate for something like `from overwatch.processing.trending.objects import mean as trendingMean` as the convention for importing trending objects